.github/workflows/ci.yml

```diff
@@ -44,11 +44,23 @@ jobs:
         export GOBIN=$HOME/bin
         go get sigs.k8s.io/krew/cmd/validate-krew-manifest@${KREW_VERSION}
         
-    - name: Validate updated plugin manifests
+    - name: Validate updated plugin manifests PUSH case
+      if: github.event_name == 'push'
```

Comment:
We need plugin validation to run only in PRs in all honesty.

Once merged, it’s too late to check manifests as we can’t act on it. Am i right @corneliusweig ?

Reply:
> Thanks. I am currently -1 for this duplication.
> I am ~80% sure there’s a better way to do this, if not, I am questioning capabilities of Github Workflows system. Do you mind asking on their community forums if there’s a way to do this cleanly?

Sure, that was a quick solution. For those cases. But there are better options.

Reply:
I agree. The tests should only run on PR and pushes to pull requests.

The duplication between the steps does not make sense to me. I still don't understand why we don't just do 
```sh
git diff --name-only --diff-filter=AM origin/master ${{ github.sha }} -- plugins/
```

This would work fine with force push AND it would _always_ run the test for everything that is about to be merged into master. Doesn't that sound right?

Reply:
Yeah. And travis was gracious enough to provide the right value as env vars.

Reply:
> I agree. The tests should only run on PR and pushes to pull requests.
> 
> The duplication between the steps does not make sense to me. I still don't understand why we don't just do
> 
> ```shell
> git diff --name-only --diff-filter=AM origin/master ${{ github.sha }} -- plugins/
> ```
> 
> This would work fine with force push AND it would _always_ run the checks for everything that is about to be merged into master. Doesn't that sound right?

In PR event it is the better option. But for push event, I needed a hash to compare. Now that it's just PR it's simple.